README.md

```diff
@@ -38,24 +38,13 @@ Compare the output to `requirements.txt`.
 
 `docker-compose` is being phased out by Docker and you may experience problems installing it via `pip`.
 
-If this happens to you, try doing the following:
+If this happens to you, use the submodule included with this repository.
 
-Clone the Docker Compose Git repository:
+Make sure the submodules are up to date and install the repo as a pip package:
 ```bash
-git clone https://github.com/docker/compose
-```
-
-Check out the latest tag which was still using the Python implementation:
-```bash
-cd compose
-git checkout 1.29.2
-```
-
-At this point, you can make the modifications needed to fix any problems you may encounter.
-
-Once done, back out and pip install the local directory:
-```bash
-cd -
+git submodule init
+# If you've already done this, make sure it's updated:
+git submodule update
```

Comment:
What does `this` refer to on L46?

Reply:
I've updated the wording to make this clear.